sqlite/migrations/0006_migrate_replications_foreign_key.up.sql

```diff
@@ -0,0 +1,34 @@
+PRAGMA foreign_keys=off;
+
+-- Removes the "ON DELETE CASCADE" from the foreign key constraint
+ALTER TABLE replications RENAME TO _replications_old;
+
+CREATE TABLE replications
```

Comment:
Are there any foreign keys pointing at the replications table that need to be updated with this dance? If no, I think this is good to go.

Reply:
No, there are no foreign keys pointed towards replications